modules/ROOT/pages/ai-gemini.adoc

```diff
@@ -0,0 +1,239 @@
+:description: A guide for integrating {pluginname} plugin using Google Gemini.
+:keywords: example, demo, custom, plugin, ai, assistant, guide, google, gemini, vertexai, ai-studio, console, gcp, platform, cloud
+
+include::partial$misc/admon-ai-pricing.adoc[]
+
+== Introduction
+
+This guide provides instructions for integrating the {pluginname} plugin using Google Gemini in {productname}.
+
+There are two version of the Google Gemini API: https://ai.google.dev/gemini-api/docs/get-started/rest[Google AI Studio] and https://cloud.google.com/vertex-ai[Google Cloud - Vertex AI]. The Google AI Studio API is ideal for quickly integrating the Google Gemini models into an application with the minimal setup required. The Google Cloud - Vertex AI API is more powerful and flexible than Google AI Studio, but requires more setup to integrate the Google Gemini models into an application.
+
+In this example, we will demonstrate how to integrate the {pluginname} plugin with the Vertex AI APIs, but the same principles can be applied to the Google AI Studio APIs by changing the API endpoint, request body, and authentication method.
+
+Vertex AI provides a `systemInstruction` field in the request body that allows controlling the output of the AI model. Google AI Studio does not currently provide this option, so the system messages would need to be provided as part of the user prompt for that use case.
+
+See https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/gemini[Google Cloud - Vertex AI Gemini API docs] for more information on the Vertex AI Gemini API.
+
+To learn more about the difference between string and streaming responses, see xref:ai.adoc#the-respondwith-object[The `respondWith` object] on the plugin page.
+
+== Authentication
+
+The access token is used in the Authorization header to authenticate the request to the Google Cloud API. The access token should be stored securely and not exposed to the client side integration. A recommended approach is to use a proxy server to generate the access token and make the API calls. For more information, see the xref:ai-proxy.adoc[AI Proxy Server reference guide].
+
+The https://www.npmjs.com/package/google-auth-library[google-auth-library] can be used to programmatically generate an access token for the Google Cloud API. Alternatively, the https://cloud.google.com/nodejs/docs/reference/vertexai/latest[Vertex AI Node.js SDK] or other client library can be used, but may require modifications to the code.
+
+== Prerequisites
+
+Before you begin, you need to have the following:
+
+* A Google Cloud Platform (GCP) account.
+* A Google Cloud Project with the Vertex AI API enabled.
+* The Project ID and Region of the Google Cloud Project.
+* An Access Token for the Vertex AI API.
+
+== String response
+
+This example demonstrates how to integrate the {pluginname} plugin with the Vertex AI API to generate string responses.
+
+include::partial$misc/admon-ai-proxy.adoc[]
```

Comment:
```suggestion

include::partial$misc/admon-ai-proxy.adoc[]

== String response

This example demonstrates how to integrate the {pluginname} plugin with the Vertex AI API to generate string responses.
```
We will only need to include this partial once per page, before the example headings.